smart-table-module/js/Table.js

```diff
@@ -176,11 +185,15 @@
 
             /**
              * insert a new column in scope.collection at index or push at the end if no index
+             * call setDefaultSortColumn to set lastCoulmnSort to already sorted column
              * @param columnConfig column configuration used to instantiate the new Column
              * @param index where to insert the column (at the end if not specified)
              */
             this.insertColumn = function (columnConfig, index) {
                 var column = new Column(columnConfig);
+                if (angular.isDefined(column.reverse)) {
```

Comment:
Hey @2013gang , `column.reverse` if not defined during column configuration would be *undefined*, otherwise the possible values can be either ***true*** or ***false***
example of such column configuration: _{label: $filter('transcribe')('Name'), map: 'name', **reverse: true**}_
Yes, `angular.isDefined` is robust enough to serve the purpose here